search/search-client-query-transformer/src/main/java/io/camunda/search/clients/PersistentWebSessionSearchClientImpl.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.search.clients;
+
+import io.camunda.search.clients.core.SearchDeleteRequest;
+import io.camunda.search.clients.core.SearchGetRequest;
+import io.camunda.search.clients.core.SearchIndexRequest;
+import io.camunda.search.clients.core.SearchQueryRequest;
+import io.camunda.search.entities.PersistentWebSessionEntity;
+import io.camunda.webapps.schema.descriptors.usermanagement.index.PersistentWebSessionIndexDescriptor;
+import java.util.List;
+
+public class PersistentWebSessionSearchClientImpl implements PersistentSessionSearchClient {
```

Comment:
This client does CRUD, it is not only writing. I will keep it as it is.

Reply:
Sorry, I don't get the point. I renamed the class to `PersistentWebSessionClient` that abstracts the underlying storage (let it be later RDBMS or Elasticsearch or whatever). I think it should stay that specific, there is not much value in making the name more general for currently not know use cases. Secondly, that's the current approach that we have "clients" per entity, instead of having one general client for everything. That way, we again separate concerns and keep the code simple and testable.